terraform/router/local/main.tf

```diff
@@ -1,33 +1,11 @@
-locals {
-  tags = {
-    System = "convox"
-    Rack   = var.name
-  }
-}
-
-module "nginx" {
-  source = "../nginx"
-
-  providers = {
-    kubernetes = kubernetes
-  }
-
-  nginx_image        = var.nginx_image
-  nginx_user         = "33"
-  namespace          = var.namespace
-  rack               = var.name
-  replicas_min       = 1
-  set_priority_class = false
-}
-
 resource "kubernetes_service" "router" {
   metadata {
     namespace = var.namespace
     name      = "router"
   }
 
   spec {
-    type = var.platform == "Linux" ? "ClusterIP" : "LoadBalancer"
+    type = "ClusterIP"
```

Comment:
@yuriadams this might cause problems for MacOS users. Let us know in your tests

Reply:
hey @heronrs, probably yes, I've just sent this to @nightfury1204. We have the same for `resolver` resource.